coding-the-legislation/40_legislation_evolutions.md

```diff
@@ -1,10 +1,11 @@
 # Legislation evolutions
 
-Openfisca handles the fact that the legislation change over time.
+Openfisca handles the fact that the legislation changes over time.
 
 ## Parameter evolution
 
-Many legislation parameters are regularly re-evaluated. In that case, formulas usually don't need to be modified: adding the new parameter value in the parameter file is enough.
+Many legislation parameters are regularly re-evaluated. 
+In that case, formulas usually don't need to be modified: adding the new parameter value in the parameter file is enough.
```

Comment:
This is out of the scope of this PR, which focus on updating the doc on formula evolutions.
I don't even know why this line break was added, making this part appear in the diff.

I'd suggest that you open an issue on the doc repo, with your suggestion so that it doesn't get lost, but it won't be treated here 🙂 .